changelogs/fragments/880-Functional_tso_command_test_cases_currently_can_not_be_run_concurrently.yml

```diff
@@ -0,0 +1,4 @@
+trivial:
+- zos_tso_command - Test cases were running in a specific sequence to validated.
+  Change summary the sequence into one test case.
```

Comment:
Perhaps reword the message to something like:
- zos_tso_command - Test suite was set up to run sequentially such that certain tests relied on prior test cases. The new changes combine those inter-dependent test cases into a single test case so that each individual test case can now be run stand-alone.